settings.gradle

```diff
@@ -4,9 +4,10 @@ ext.buildCachePsw = System.getenv("GRADLE_BUILDCACHE_PSW")
 
 buildCache {
     local {
-        enabled = !isCiServer
+        enabled = !(buildCachePsw?.trim() && buildCacheUser?.trim())
```

Comment:
It's really important to have local cache enabled when running on a fork, where the remote cache is not available. This enables local cache if remote cache username/password is not available.